src/opers.c

```diff
@@ -1946,6 +1946,9 @@ static ALWAYS_INLINE Obj GetMethodUncached(
         // entry n+3 is the rank
         // entry n+4 is the info text
         // entry n+5 is, if set, the location where the method was installed
+        // entry n+6 is, if set, the relative rank that was supplied when
+        //                       the method was installed, either as a small integer or a function
+        //                       of no arguments
```

Comment:
That's a somewhat odd wrapping (looking at the differing line lengths); also, could perhaps align at "if set"?

And perhaps should say: "a function of no arguments, which returns a small integer to be used as rank"?

This use of such a function as "base rank" should ideally be documented somewhere, too, no? And I guess we'd also have to go through the library to search for places where this should be used...